Configurations/10-main.conf

```diff
@@ -1317,9 +1329,12 @@ my %targets = (
     "VC-common" => {
         inherit_from     => [ "BASE_Windows" ],
         template         => 1,
-        cc               => "cl",
-        cpp              => '$(CC) /EP /C',
-        cflags           => "-W3 -wd4090 -Gs0 -GF -Gy -nologo",
+        CC               => "cl",
+        CPP              => '$(CC) /EP /C',
+        CFLAGS           => "-W3 -wd4090 -nologo",
+        COUTFLAG         => "/Fo",
```

Comment:
Actually with earlier mentioned contradiction between overridable CC and non-overridable cflags, I'd like to argue that COUTFLAG should be non-overridable.

Reply:
And this applies even to ASOUTFLAG.  [And maybe all \*OUTFLAG?] To reiterate. User is not actually free to override CC or AS with arbitrary value. Replacement has to be flag-compatible with default tool, at least a subset of flags has to be. And what I'm trying to say is that outflags are part of the said subset.